commit 892c05c093858086d808aeb366b2e11106dd96c6
Author: Willy Tarreau <w@1wt.eu>
Date:   Sun Feb 20 11:43:07 2011 +0100

    rtlwifi: Let rtlwifi build when PCI is not enabled
    
    On systems where PCI does not exist, a build of rtlwifi will fail.
    Apply the same fix in case there are systems with PCI but not USB.
    
    Signed-off-by: Willy Tarreau <w@1wt.eu>
    Signed-off-by: Larry Finger <Larry.Finger@lwfinger.net>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/rtlwifi/Makefile b/drivers/net/wireless/rtlwifi/Makefile
index c3e83a1..52be12e 100644
--- a/drivers/net/wireless/rtlwifi/Makefile
+++ b/drivers/net/wireless/rtlwifi/Makefile
@@ -5,9 +5,12 @@ rtlwifi-objs	:=		\
 		core.o		\
 		debug.o		\
 		efuse.o		\
-		pci.o		\
 		ps.o		\
 		rc.o		\
 		regd.o
 
+ifneq ($(CONFIG_PCI),)
+rtlwifi-objs	+= pci.o
+endif
+
 obj-$(CONFIG_RTL8192CE)		+= rtl8192ce/
